contrib/mpi-proxy-split/lower-half/gethostbyname-static/gethostbyname_static.c

```diff
@@ -174,15 +175,16 @@ int getaddrinfo(const char *__restrict node,
   } else if (childpid > 0) {
     if (hints != NULL) {
       writeall(pipefdin[1], hints, sizeof(*hints));
-      writeall(pipefdin[1], hints->ai_addr, sizeof(*(hints->ai_addr)));
+      // hints->ai_addr != NULL on input; no need to send *(hints->ai_addr)
```

Comment:
typo? `hints->ai_addr == NULL`

Reply:
Very good point!  I've fixed this now.  The comment used to be `hinst->ai_addr != NULL`, and you're correct.  I meant to write `==`.  So, you're even good at finding bugs inside my comments, huh?  :-)